### PR TITLE
Feature/sign up component

### PR DESCRIPTION
**Pull request by amxra**
#### What does this PR do?

-  Display the sign up component

#### Description of Task to be completed?
- Built out the full responsive UI for the signup page

#### How should this be manually tested?
- run npm start to view in localhost

#### What are the relevant Trello stories?
- https://trello.com/c/o7nydZaP/34-sign-up-to-get-a-personal-profile

#### Relevant Screenshots (if appropriate)
- None

#### Questions 
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Easybuoy — APPROVED** [review]
Nice work @amxra

File: public/index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
     <link rel="manifest" href="/site.webmanifest" />
-    <title>React App</title>
+    <title>TrackFit</title>
```
Comment (melquip) [inline public/index.html]: The name is trackdrills :)

**Review by Leelsmuth — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (Easybuoy) [reply public/index.html]: Yeah true, nice catch @melquip

File: src/components/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import styled from "styled-components";
+import { Button } from "@chakra-ui/core";
+import banner from "../assets/banner.jpg";
+
+const SignUpStyle = styled.div`
```
Comment (melquip) [inline src/components/SignUp.jsx]: Not sure, but maybe we should split styles and components into different files. Something to ask the TL.

**Review by melquip — APPROVED** [review]
Everything's looking good 👍

File: src/components/SignUp.jsx (hunk 2 above)
Comment (Easybuoy) [reply src/components/SignUp.jsx]: Yes, let's talk about it at End of Day wrapup

**Review by Leelsmuth — APPROVED** [review]

**Review by Leelsmuth — APPROVED** [review]

**Pull request merged**